### DISCUSSION TITLE
How to update existed elsa workflow 1.0 to elsa workflow 2.0?

### DISCUSSION THREAD

**Discussion by 1002527441** [elsa-workflows/elsa-core#843] [Q&A]
Is there a guide or manual ?

**Comment by sfmskywalker** [comment]
Not at the moment I'm afraid. I'm still pushing hard on getting Elsa 2 ready for release, after which I'll focus on documentation. I will then probably also add some notes about the differences between v1 & v2 which might help understand what one needs to do to upgrade. When it comes to workflow definitions, you will have to manually recreate them. Which might be OK if you have only a few small workflows. But if you have many & complicated workflows,  you might want to create a migration tool that transforms the v1 structure to that of v2.

**Reply by 1002527441** [reply]
thanks for your reply, and thanks for creating the great tools.
